css/types/angle-percentage.json

```diff
@@ -25,22 +25,22 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "14"
             },
             "safari": {
               "version_added": "4"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "3.2"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "2"
```

Comment:
I've wondered many times if we needed something to indicate whether this value is a Chromium number or an Android number. This illustrates why.

Reply:
I can see your point. A hint what it is, is somewhat recorded in our browser data, though: https://github.com/mdn/browser-compat-data/blob/master/browsers/webview_android.json#L26

Reply:
I take this to meant that this case is a mapping from Chrome 2's WebKit engine number to Android 2's engine number?

Reply:
Yes! :)